doc/targets.txt

```diff
@@ -183,21 +183,30 @@ Supported trigger characters:
     "         (work on double quotes)
     `         (work on back ticks)
 
-The following examples will use single quotes, but they all work for each
-listed quoting character accordingly.
+These quote text objects try to be smarter than the default ones. They count
+the quotation marks from the beginning of the line to decide which of these
+are the beginning of a quote and which ones are the end.
+
+If you type `ci,` on the `,` in the example below, it will automatically skip
```

Comment:
Thanks! 

The one below might be more clear like this:
>`", "` between `"hello"` and `"world"`

If you agree you can do that in #155 too.